Classes/PHPExcel/Reader/OOCalc.php

```diff
@@ -535,7 +535,7 @@ public function loadIntoExisting($pFilename, PHPExcel $objPHPExcel)
                                             array_push($dataArray, $pData);
                                         }
                                     }
-                                    $allCellDataText = implode($dataArray, "\n");
+                                    $allCellDataText = implode("\n", $dataArray);
```

Comment:
Das wird uns in unserem Code auch um die Ohren fliegen. Konnte man das automatisieren mit Rector PHP?